Paskab012/cartix-web1
# Toggle new

#### Hello Reviewer ✋🏽 
## Description of changes made ♻️
Fix GitHub action deployment to firebase hosting

#### Type of change

> **Please select the relevant options**

- [ ] Breaking change (fix or a feature that would cause an existing feature not to work as expected)

#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/Paskab012/cartix-web1.git)
- Check out this branch by `toggle-new`
- Run `yarn or npm install to install the project dependencies
- Run `npm run start:cartix` to start the development server

#### Checklist


#### Screenshots

|        BEFORE                  |                     AFTER       |
| :----------------------------: | :-----------------------------: |
| CI break => dependency missing | CLI build passes                |
